File: 02.PYTHON/pg-166-169/pg_166_12.py
print("");
print("PYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPY");
print("READ TWO TWO-DIGIT INTEGERS AND DETERMINE IF THEY BOTH HAVE DIGITS IN COMMON");
print("PYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPY");
print("");

dig1 = 0;
dig2 = 0;
dig3 = 0;
dig4 = 0;
num1 = int(input("Write the first digit: "));
num2 = int(input("Write the second digit: "));

if(num1 < 0):
    num1 *= -1;

if(num2 < 0):
    num2 *= -1;

if(num1 >= 10 and num1 <= 99 and num2 >= 10 and num2 <= 99):
    dig2 = num1 % 10;
    dig1 = num1 // 10;
    dig4 = num2 % 10;
    dig3 = num2 // 10;

    if(dig1 == dig3 or dig1 == dig4 or dig2 == dig3 or dig2 == dig4):
        print("There are digits in common between both writen numbers.");
    else:
        print("There are not digits in common between both writen numbers.");

else:
    print("Some of the writen numbers doesn't have two digits. Please try again!");

File: 01.JAVA/pg-166-169/pg_166_12.java
import java.util.Scanner;

public class pg_166_12
{
    public static void main(String[] args)
        {
            Scanner enter = new Scanner(System.in);
            int num1, num2, dig1, dig2, dig3, dig4;

            System.out.println("");
            System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJV");
            System.out.println("READ TWO TWO-DIGIT INTEGERS AND DETERMINE IF THEY BOTH HAVE DIGITS IN COMMON");
            System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJV");
            System.out.println("");

            System.out.print("Write the first integer: ");
            num1 = enter.nextInt();

            System.out.print("Write the second integer: ");
            num2 = enter.nextInt();

            if(num1 < 0)
            {
                num1 *= -1;
            }
            if(num2 < 0)
            {
                num2 *= -1;
            }

            if(num1 >= 10 && num1 <= 99 && num2 >= 10 && num2 <= 99)
            {
                dig4 = num2 % 10;
                dig3 = num2 / 10;
                dig2 = num1 % 10;
                dig1 = num1 / 10;

                if(dig1 == dig3 || dig1 == dig4 || dig2 == dig3 || dig2 == dig4)
                {
                    System.out.println("There are digits in common.");
                }
                else
                    {
                        System.out.println("There are not digits in common.");
                    }
            }
            else
                {
                    System.out.println("Some of written numbers doesn't have two digits. Please try again!");
                }
        }
    
}